src/App.jsx

```diff
@@ -15,7 +15,7 @@ function App() {
           <NavBar className="nav-bar" />
           <Routes>
             <Route path="/articles" element={<AllArticles />} />
-            <Route path="/articles/:article_id" element={<Article />} />
+            <Route path="/articles/:article_id/*" element={<Article />} />
```

Comment:
Not sure why this was updated. This endpoint should be `/articles/:article_id` without anything else as that is the endpoint we are wanting to work with.